lib/ui/shared/smart_widgets/thread_card/widgets/thread_card_main.dart

```diff
@@ -64,17 +66,28 @@ class ThreadCardMain extends ViewModelWidget<ThreadCardViewModel> {
                         ],
                       ),
                       SizedBox(height: 10),
-                      RichText(text: userPost!.message),
+                      // TextParser(
+                      //     "[@michael:51515151] Hello this is an example of the ParsedText, links like http://www.google.com or http://www.facebook.com are clickable and phone number 444-555-6666 can call too. But you can also do more with this package, for example Bob will change style and David too. foo@gmail.com And the magic number is 42! #react #react-native"),
+                      TextParser(userPost!.message),
                     ],
                   ),
-                ),
-                SizedBox(height: 10),
-                EmojisList(userPost: userPost),
-                SizedBox(height: 15)
-              ],
-            ),
-          )
-        ],
+                  SizedBox(height: 10),
+                  EmojisList(userPost: userPost),
+                  SizedBox(height: 15),
+                  userPost!.userThreadPosts != null
```

Comment:
I just notice this logic is used in the view, it should not, the value should be gotten from the viewmodel